cnf-tests/README.md

```diff
@@ -446,6 +447,16 @@ To make it quicker, a `CLEAN_PERFORMANCE_PROFILE="false"` can be set to instruct
 docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e CLEAN_PERFORMANCE_PROFILE="false" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ```
 
+## Running in single node cluster
+
+Running tests on a single node cluster causes some limitations to be imposed:
+- longer timeouts for certain tests
+- some tests requiring multiple nodes are skipped
+
+The longer timeouts concern mostly SRIOV and SCTP tests. Reconfiguration requiring node reboots cause a reboot of the whole environment and hence takes longer to complete.
```

Comment:
whole environment including the control openshift control plane

Reply:
Done

Reply:
I don't see a change in the PR